app/views/layouts/application.html.erb

```diff
@@ -8,7 +8,17 @@
 </head>
 <body>
 
-<%= yield %>
+  <%= render partial: "shared/header" %>
+
+  <% flash.each do |name, msg| %>
+    <%= content_tag :div, msg, class: "#{name} alert #{flash_check(name)}" %>
+  <% end %>
+
+  <div class="row">
+    <div class="col-md-10 col-md-offset-1">
+      <%= yield %>
+    </div>
+  </div>
```

Comment:
I did this because the forms were hugging the sides and wanted a bit of space between our contents and the edge, feel free to change the styling and/or we can change the styling later.